.github/workflows/community-label.yml

```
@@ -1,10 +1,10 @@
 name: Add Community Label
 
 on:
+  pull_request:
```

```suggestion
  pull_request_target:
    types: [opened]
```

What's the reason for this change?

I believe we'll need to use `pull_request_target` in order to have permissions for community contributors to add the label (but, I might be wrong on that one)... and do we need to run this on types other than `opened`? It seems like doing some backfilling might be better than running this action for other states of a PR, as one's community status is unlikely to change in the lifecycle of a PR.

---

@jonathansadowski I woke up last night and thought "damn I forgot to change this back" lol. It was just for testing the action. Will fix.